docs/src/pages/guides/FHIRClinicalReasoningGuide.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+
```

Comment:
You also need to update `docs/src/data/nav-items.yaml` if you want it to show up in the navigation on the left of the site.